Packs/ShadowxSOCAI/Integrations/ShadowxSOCAI/ShadowxSOCAI_description.md

```diff
@@ -0,0 +1,59 @@
+# ShadowX SOCAI
+
+ShadowX SOCAI  , filter sensitive data in security logs and analyze with the GEN AI solutions like  Chatgpt,Gemini,Claude  and produces enriched guidance (response, recommendation, risk/result, prediction score).
+
+## Authentication Modes
+
+### API Key (recommended)
+- Set **API Key** in the instance.
+- By default we POST JSON to `/api/FreeTextSearch/ShadowxProxy` with `Authorization: Bearer <API_KEY>`.
```

Comment:
For a more formal and clear tone, please replace "we" with "the integration" on lines 9 and 10.